### PR TITLE
Added a fullscreen preview mode

### LINKED ISSUE #1242: Full-screen preview without publishing
Watching the tweets for National Girls Learning Code day, I noticed this pattern happening a lot:

image

Here you see someone wanting to either see or show their work without the editor or file tree.  In Bramble we actually have code to do this, and open the preview in another tab.  I think we need a button to make this easy to do.  We might tell people to just publish their work, but if you're working anonymously, this isn't possible.

cc @flukeout

### PR DESCRIPTION
**Pull request by flukeout**
Closes #1242 

Works by adding a class to both the thimble and brackets UI restyles things to show only the Brackets preview. Also overlays some of the Preview bar on top so you can get out of the fullscreen preview.

Related brackets PR incoming soon.

### CODE REVIEW

**Comment by flukeout** [comment]
Pairs nicely in https://github.com/humphd/brackets/pull/504

**Comment by flukeout** [comment]
@humphd R?

File: public/editor/scripts/editor/js/fc/bramble-ui-bridge.js [hunk 1]
```diff
@@ -222,6 +222,22 @@ define(function(require) {
       }
     }
 
+    $(".fullscreen-preview-toggle").click(function() {
+      toggleFullscreenPreview();
+    });
+
+    function toggleFullscreenPreview(){
+      if($("body").hasClass("fullscreen-preview")){
+        analytics.event("Fullscreen Off");
```
Comment (humphd) [inline public/editor/scripts/editor/js/fc/bramble-ui-bridge.js]: The rest of the analytics events have no spaces in the name--not sure if that matters or not, but let's be consistent: `FullscreenOn` and `FullscreenOff`.

File: public/editor/stylesheets/editor.less [hunk 2]
```diff
@@ -787,3 +813,70 @@
...
+      box-sizing: border-box;
+      min-width: 0;
+      border-radius: 1px;
+
+      .nav-container {
+        min-width: 0;
+        padding: 0 12px;
+
+        .pull-right {
+          margin-left: 0px;
```
Comment (humphd) [inline public/editor/stylesheets/editor.less]: `0` with no `px`

File: public/editor/stylesheets/editor.less [hunk 3]
```diff
@@ -787,3 +813,70 @@
...
+        }
+      }
+    }
+
+    #preview-pane-nav-refresh {
+      margin: 0 10px 0 0;
+    }
+
+    .fullscreen-preview-toggle {
+      margin-left: 0px;
```
Comment (humphd) [inline public/editor/stylesheets/editor.less]: Same here, drop `px`

**Comment by humphd** [comment]
A few nits, and let's update this to match comments I made in your Bramble patch about splitting that toggle method into enable/disable.

r=me after all that.

**Comment by flukeout** [comment]
@humphd Made the changes, R?

**Comment by humphd** [comment]
Yeah, looks good.  Did you also update Brackets?  We should land that first.

r+

**Comment by flukeout** [comment]
@humphd updating the brackets PR now

**Pull request merged**